src/less/properties/base/badge-properties.less

```diff
@@ -1,4 +1,8 @@
-.badge {
-    --badge-background-color: @badge-background-color;
-    --badge-foreground-color: @badge-foreground-color;
+@media (prefers-color-scheme: dark) {
+    .skin-experiment-dark-mode {
+        .badge {
+            --badge-background-color: @color-r4-dark-mode;
+            --badge-foreground-color: @color-background-default-dark-mode;
+        }
+    }
 }
```

Comment:
I think it's good to have them both combined and I'm fine with this, but then my question is, why not just include this in the `badge.less` file since it's basically CSS overrides? 
Why do we need the property file anyways other than just to simplify the imports?
We sometimes might run into some issues with our preprocessor rearranging `@media` queries and reordering them which is another advantage of having them in the main css file.

Reply:
Yep, totally agree. I was trying to do this in phases to keep the diff readable, but now that you mention it, moving them to the base less file and deleting all the properties would be a straightforward diff too. Let me think on it...